project.yml

```diff
@@ -3,6 +3,9 @@ name: IronRoadForChildren
 fileGroups:
   - project.yml
   - README.md
+  - Gemfile
+  - Gemfile.lock
```

Comment:
No need to change, but as the `Gemfile.lock` should not be touched, I would not include it in the Xcode filetree.

Reply:
For Version checking it should be accessible, so I would leave it in :D But good point!